public/app/features/search/page/components/SearchView.tsx

```diff
@@ -84,6 +91,34 @@ export const SearchView = ({ showManage, folderDTO, queryText, hidePseudoFolders
     [searchSelection]
   );
 
+  const keyListener = useCallback((evt: KeyboardEvent) => {
+    switch (evt.code) {
+      case 'ArrowDown': {
+        highlightIndexRef.current++;
+        setHighlightIndex(highlightIndexRef.current);
+        break;
+      }
+      case 'ArrowUp':
+        highlightIndexRef.current = Math.max(0, highlightIndexRef.current - 1);
+        setHighlightIndex(highlightIndexRef.current);
+        break;
+      case 'Enter':
+        if (highlightIndexRef.current > 0 && responserRef.current) {
+          const url = responserRef.current.view?.fields?.url?.values?.get(highlightIndexRef.current - 1);
+          if (url) {
+            locationService.replace(url);
+          }
+        }
+    }
+  }, []);
+
+  useEffect(() => {
+    document.addEventListener('keydown', keyListener);
+    return () => {
+      document.removeEventListener('keydown', keyListener);
+    };
+  }, [keyListener]);
```

Comment:
The keyboard arrow navigation should only be active when the keyboard have focus. Having it here  could conflict with other selects and components in the SearchView